test/uk/gov/hmrc/tai/model/domain/EmploymentBuilderSpec.scala

```diff
@@ -260,7 +259,7 @@ class EmploymentBuilderSpec extends PlaySpec with MockitoSugar {
 
         unifiedEmployments.size mustBe 2
 
-        verify(mockAuditor, times(0)).sendDataEvent(meq(auditTransactionName), any())(any())
+        verify(mockAuditor, times(1)).sendDataEvent(meq(auditTransactionName), any())(any())
```

Comment:
Why has this changed?

Reply:
Since we have given 3 AnnualAccounts with Sequence number  0,1,2 and only 2 matches with Employment with sequence number 1 and 2. So for one of the Annual account auditAssociatedEmployment gets invoked in combineAccountsWithEmployments. We can revert this back to 0 times invocation if we provide 2 matching Annual Accounts. Let me know if we want to revert it back to 0 times invocation.

Reply:
Yes please, lets turn it back to 0

Reply:
made change in test and reverted back to 0, Done